.gitignore

```diff
@@ -7,3 +7,5 @@ __pycache__/
 *.out
 htmlcov/
 .coverage
+/.vs
+/cve_bin_tool/strings.c
```

Comment:
I think line 11 can be omitted (it's probably useful to you right now but could be weird and surprising later)

Reply:
Yes, I'm working on writing `string` and `file` in C. I think I could finish it by tomorrow. Then I could update this pr.